src/profile-logic/process-profile.js

```diff
@@ -541,18 +541,23 @@ export function serializeProfile(profile) {
 
 function unserializeProfile(profile) {
```

Comment:
in latest master, we have a TODO as a comment to this function, please remove this as well :)